02_r-basics.Rmd

```diff
@@ -4,7 +4,7 @@
 source("common.R")
 ```
 
-<!--Original content:http://stat545.com/block002_hello-r-workspace-wd-project.html-->
+<!--Original content: http://stat545.com/block002_hello-r-workspace-wd-project.html-->
```

Comment:
Is there any practical significance to these being http vs https? Because it seems to not be consistently one or the other in the diffs here.

Reply:
There isn't, I opened a new PR that makes them all https.